space/spherical.glsl

```diff
@@ -0,0 +1,17 @@
+/*
+author: Kathy McGuiness
+description: Returns the spherical coordinates
+use: sphericalSDF(<vec3> st)
+*/
+
+#ifndef FNC_SPHERICAL
+#define FNC_SPHERICAL
+vec3 spherical( in vec3 st) 
```

Comment:
this is spherical to cartesian? should be good to specify from what to what type of coordenates

Reply:
yes, will edit

Reply:
let's re use cart2polar but overload it so together with the `vec2 cart2polar(vec2)` we have this one with `vec3 cart2polar(vec3)`